book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Please add getters and setters (attr_accessor) for @rentals, not just getters (attr_reader).

Reply:
@Omar-Muhamad please can you elaborate because I don't understand what you expect me to do here.